atomics/include/desul/atomics/Generic.hpp

```diff
@@ -382,7 +300,7 @@ atomic_oper_fetch(const Oper& op,
   }
   return return_val;
 #else
-  static_assert(false, "Unimplemented lock based attomic\n");
+  static_assert(false, "Unimplemented lock based atomic\n");
```

Comment:
You missed that one
https://github.com/desul/desul/blob/7d3b455a260f492376d0fdf9270e6cfd5c8038bf/atomics/include/desul/atomics/Generic.hpp#L255